identity-service/src/notifications/index.js

```diff
@@ -110,15 +132,19 @@ class NotificationProcessor {
       logger.info('processEmailNotifications')
       await processEmailNotifications(expressApp, audiusLibs)
       await processDownloadAppEmail(expressApp, audiusLibs)
+
+      // Wait 10 minutes before re-running the job
+      await new Promise(resolve => setTimeout(resolve, 10 * 60 * 1000))
+      await this.emailQueue.add({ type: unreadEmailJob }, { jobId: `${unreadEmailJob}:${Date.now()}` })
```

Comment:
this needs to be defined like announcementJobType?

Reply:
The first param is the job data which is actually not used at all. 
The second parameter used to pass in a unique jobID - for which the prefix shouldn't matter b/c of the date.now.